### PR TITLE
Feat: Added Results Page

### PR DESCRIPTION
**Pull request by shuntzhere**
Screenshot (32)

### CODE REVIEW

File: css/category.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: center;
+  padding: 1rem 0;
+  font-family: "Palanquin Dark", sans-serif;
+}
+
+.category-list {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (rittik-ghosh) [inline css/category.css:19]: can use utility classes for these.
Due to time constraints it can be refactored later

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  --bg-reef: #00f7ff;
+  --bg-dark: #4592af;
+
+  --color-white: #fdfdfd;
+  --color-black: #222222;
+  --color-shadow: #cfd9de;
+  /* font families */
+  /* font-family: "Lora", serif;
+  font-family: "Hind Siliguri", sans-serif;
+  font-family: "Palanquin Dark", sans-serif;
```
Comment (rittik-ghosh) [inline css/main.css:21]: can rewove the comments

File: rules.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  <body>
+    <div class="rules-container">
+      <nav class="quiz-navbar flex flex-align-center justify-between">
+        <div class="nav-logo flex justify-center">gehraaiyaan</div>
+        <div class="nav-right">
+          <!-- <button class="btn">
+            <p>Signup</p>
+            <div>
+              <img class="svg" src="/assets/svg/angle-right-solid.svg" />
+            </div>
```
Comment (rittik-ghosh) [inline rules.html:19]: remove the commented code

**Review by rittik-ghosh — COMMENTED** [review]
Have suggested some changes. Rest all looks good

File: css/category.css (hunk 1 above)
Comment (shuntzhere) [reply css/category.css:19]: Yes, will do that will next fix.

**Review by Sameony — APPROVED** [review]
structured and orderly code. you can merge it

File: rules.html (hunk 3 above)
Comment (shuntzhere) [reply rules.html:19]: Yes will do that.

File: css/main.css (hunk 2 above)
Comment (shuntzhere) [reply css/main.css:21]: Yes will be doing that.

**Pull request merged**